practice-two/src/js/views/product.js

```diff
@@ -167,4 +168,69 @@ export default class ProductView {
 
   }
 
+  bindToggleForm() {
+    const mainContent = document.querySelector('.main-content');
+
+    const formProductHTML = `
+      <div class="modal-overlay hidden">
+        <form action="" method="post" class="add-product-container">
+          <h2 class="add-product-title">Add products</h2>
```

Comment:
This form should be used for bold add and edit product, so please think more about the common naming for it